### ISSUE TITLE
TAPAS finetune.

### ISSUE THREAD

**Issue by lairikeqiA** [google-research/tapas#51]
When TAPAS is fine-tuned, are the parameters of Bert fixed?

**Comment by thomasmueller-google** [comment]
No, every parameter is tuned.

**Comment by lairikeqiA** [comment]
If I fix the parameters of Bert when TAPAS is fine-tuned,  how the dev accuracy should be on WTQ?

**Comment by thomasmueller-google** [comment]
No idea, but probably very low since most of the parameters are in the Bert model (there is only one additional matrix for the token logits).

**Comment by lairikeqiA** [comment]
Thanks!
